src/style.css

```diff
@@ -0,0 +1,93 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.box {
+  display: flex;
+  flex-basis: 100%;
+  justify-content: center;
+  align-items: center;
+  font-size: 100px;
+  font-family: Arial, Helvetica, sans-serif;
+  color: #fff;
+  min-height: 300px;
+  background-color: #f00;
+}
+
+.box-2 {
+  background-color: #c00;
+}
+
+.box-3 {
+  background-color: #900;
+}
+
+.box-4 {
+  background-color: #600;
+}
+
+.box-5 {
+  background-color: #300;
+}
+
+.box-6 {
+  background-color: #000;
+}
+
+@media(min-width: 300px) {
+  .container {
+    flex-direction: column;
+  }
+  .box {
+    flex-basis: 100%;
+  }
+}
```

Comment:
```suggestion
```

You can easily remove this media-query because you have `min-width: 600px` and `min-width: 900px` queries, so in any other cases, which differ from those two screen widths (that means <600px) will be used styles written above.
And no need to change flex-direction to column, because you use `flex-wrap:wrap` and all your blocks will be moving to the next line if they not fit the screen width